utils/generic.py

```diff
@@ -5,18 +5,20 @@
 import time
 import urllib.request
 import uuid
+from types import SimpleNamespace
 from typing import List, Union
 
+from IPython.display import HTML, display
```

Comment:
We need to update the imports to use mat3ra.api_client everywhere - maybe a seprate PR